app/src/main/java/com/gamecodeschool/asteroidsfs/GameView.java

```diff
@@ -8,60 +8,59 @@
 import android.graphics.Paint;
 import android.util.Log;
 import android.view.SurfaceHolder;
-import android.view.SurfaceView;
 import android.content.Context;
 import android.graphics.RectF;
 import android.graphics.Point;
+import android.graphics.drawable.AnimationDrawable;
+import android.widget.ImageView;
+import android.view.View;
+import android.app.Activity;
 
 import java.util.ArrayList;
 
 // make this an interface
-public class GameView{
-
-    private SurfaceHolder myHolder;
-    private Canvas myCanvas;
-    private Paint myPaint;
-    private Context ourContext;
-    Matrix shipMatrix = new Matrix();
-
-    GameView(Context context, SurfaceHolder surfHolder){
-        ourContext = context;
-        myHolder = surfHolder;
-        myPaint = new Paint();
+public class GameView extends Activity {
```

Comment:
If you're suggesting name changes. i would prefer we do something like mSurfaceHolder to signify that the surfaceholder is a member instead of just using lowercase starting name.

Reply:
that's fine i was just using what they use in other classes by convention but idc that much about it